### PR TITLE
PCC NTupler updates 74X

### PR DESCRIPTION
**Pull request by capalmer85**
Same updates as in https://github.com/cms-sw/cmssw/pull/9285 back ported for 74X

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    edm::InputTag   fPrimaryVertexCollectionLabel;
+    edm::InputTag   fPixelClusterLabel;
+    edm::InputTag   fPileUpInfoLabel;
+  
+    static const int MAX_VERTICES=200;
+
+    bool fAccessSimHitInfo;
+
+    TFile *fFile; 
+    TTree *fTree;
```
Comment (cvuosalo) [inline RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.h]: Where are fFile and fTree used?

File: RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+    }
+
+    if(sampleType=="MC"){
+        pileUpToken=consumes<std::vector< PileupSummaryInfo> >(fPileUpInfoLabel);
+        tree->Branch("nPU",&nPU,"nPU/I");
+    }
+}
+
+// ----------------------------------------------------------------------
+PCCNTupler::~PCCNTupler() { }
```
Comment (cvuosalo) [inline RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc]: Shouldn't pileup and tree be deleted here? Or are they deleted somewhere else?

**Comment by cvuosalo** [comment]
+1

For #9727 965a86b6ca821f1e01cfe1f27f7c07446a6e28b3

Updating the pixel cluster counter ntupler. There should be no affect on standard workflows.

The code changes are satisfactory. Some further code revisions might be desirable, but since this PR is a backport, they are not crucial. Jenkins [tests](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-9727/186299/summary.html) against baseline CMSSW_7_4_X_2015-06-19-2300 show no significant differences, as expected.

**Comment by capalmer85** [comment]
@cvuosalo Thank you for your comments and your "+1".  Next update I will remove the vestigial declarations and delete appropriately in the deconstructor.

**Comment by capalmer85** [comment]
Anyone know why isn't this merged? @cvuosalo, @slava77, @degano

**Comment by slava77** [comment]
@davidlange6 
the changes here are in a standalone ntuple code
It should be safe to go to 74X.
@capalmer85 Can't you use 75X release version?

**Comment by capalmer85** [comment]
Yes, I can.  I was just about to make another updated and I was checking if everything worked in 76, 75 and 74 and didn't find the package in 74.  

Btw, am I right that I should start development from 76X now?  Should I continue to backport to 75X and 74X?

**Comment by slava77** [comment]
On 8/17/15 2:58 PM, capalmer85 wrote:
> Yes, I can. I was just about to make another updated and I was checking
> if everything worked in 76, 75 and 74 and didn't find the package in 74.
> 
> Btw, am I right that I should start development from 76X now? Should I
> continue to backport to 75X and 74X?


New developments should be put in 76X first and then backported to as
far as they are needed.

**Comment by davidlange6** [comment]
+1

**Pull request merged**